kcidb/create_object.py

```diff
@@ -59,101 +59,43 @@ def get_kernel_hash(kernel_version):
 
 base_kernel_hash=str(get_kernel_hash(kernel_version).stdout).strip("\n")
 
-json_template = """
-{
-    "revisions": [
-        {
-            "id": "2c85ebc57b3e1817b6ce1a6b703928e113a90442",
-            "origin": "gkernelci",
-            "git_repository_url": "git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git",
-            "git_repository_branch": "5.8",
-            "contacts": [
-                "Mike Pagano <mpagano@gentoo.org>"
+r_id = str(base_kernel_hash) + "+" + str(patchset_hash)
+print(r_id)
+
+data = dict(
+    version=dict(
+        major=3,
+        minor=0
+    ),
+    revisions=[
+        dict(
+            id=r_id,
+            origin="gkernelci",
+            git_repository_url="git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git",
+            git_repository_branch=kernel_version,
+            contacts=[
+                "Mike Pagano <mpagano@gentoo.org>",
+                "Alice Ferrazzi <alicef@gentoo.org>"
             ],
-            "patch_mboxes": [
-                {
-                    "name": "1500_XATTR_USER_PREFIX.patch",
-                    "url": "https://gitweb.gentoo.org/proj/linux-patches.git/plain/1500_XATTR_USER_PREFIX.patch?h=5.8&id=1d996290ed3f1ffdc4ea2d9b4c4d2cf19ccc77d3"
-                },
-                {
-                    "name": "1510_fs-enable-link-security-restrictions-by-default.patch",
-                    "url": "https://gitweb.gentoo.org/proj/linux-patches.git/plain/1510_fs-enable-link-security-restrictions-by-default.patch?h=5.8&id=1d996290ed3f1ffdc4ea2d9b4c4d2cf19ccc77d3"
-                }
-            ],
-            "log_url": "https://kernel-ci.emjay-embedded.co.uk/api/v2/logs/16852/raw",
-            "valid": true
-        }
-    ],
-    "builds": [
-        {
-            "id": "gkernelci:50-6",
-            "origin": "gkernelci",
-            "revision_id": "9ece50d8a470ca7235ffd6ac0f9c5f0f201fe2c8+bf3c6a7247ae590c0d2965622961b74b6c99a92fec70d07fa4025cb6fcb944a9",
-            "architecture": "amd64",
-            "log_url": "https://kernel-ci.emjay-embedded.co.uk/api/v2/logs/16853/raw",
-            "valid": true,
-            "misc": {
-                "url": "https://kernel-ci.emjay-embedded.co.uk/#/builders/50/builds/6"
-            }
-        }
+            patch_mboxes=get_patches_list(),
+            log_url="https://kernel-ci.emjay-embedded.co.uk/api/v2/logs/16852/raw",
```

Comment:
This will get replaced with a revision-specific URL, I assume?

Reply:
revision specific log URL is it the URL that will show if the patches apply correctly or if the kernel build correctly ?

Reply:
The revision-specific URL in this case would be a log of trying to apply patches, such as the output of "git am".

Reply:
I see. I will add the URL log with the kernel patching results.